### ISSUE TITLE
Logging ENHANCEMENT

### ISSUE THREAD

**Issue by Dr-Hardware** [StrangeLoopGames/EcoIssues#2681]
•	A chainsaw or logging machine - if the excavator can mine why can't a lumberjack have a machine?

**Comment by MarjieAVolk** [comment]
Yes, we definitely want more advanced machinery for all the different resources, eventually
